### DISCUSSION TITLE
How to integrate multiple scATAC-seq dataset

### DISCUSSION THREAD

**Discussion by koh2ng0** [GreenleafLab/ArchR#1917] [Questions / Documentation]
Hi,

I have tried to integrate 3 scATAC-dataset from different studies using ArchR.
Is there a tutorial related to integrating multiple datasets using ArchR like Signac?
Or do I just proceed with processes following a standard tutorial of ArchR?
Could you give me any advice about integration scATC-seq using ArchR?

Thanks.

**Comment by rcorces** [comment]
Its not clear what you mean by integration. If you mean batch correction, then ArchR allows for use of harmony. All of this is shown in the full user manual. Please make sure to read it in full.
